File: TVMazeApp.Server/Models/Person.cs
namespace TVMazeApp.Server.Models;

public class Person
{
    public int Id { get; set; }
    public string Url { get; set; }
    public string Name { get; set; }
    public Country Country { get; set; }
    public DateTime? Birthday { get; set; }
    public object Deathday { get; set; }
    public string Gender { get; set; }
    public Image Image { get; set; }
    public int Updated { get; set; }
    public Links Links { get; set; }
}

File: tvmazeapp.client/src/entities/Person.ts
import { Country, Image, Links } from "./HelperInterfaces";

export default interface Person {
    id: number;
    url: string;
    name: string;
    country: Country;
    birthday?: Date;
    deathday?: Date;
    gender: string;
    image: Image;
    updated: number;
    links: Links;
}
